pages/screens/Home/index.tsx

```diff
@@ -0,0 +1,29 @@
+import type { NextPage } from 'next';
+import clsx from 'clsx';
+import Head from 'next/head';
+import styles from './styles.module.css';
+
+const Home: NextPage = () => {
+  return (
+    <div className={styles.container} >
+      <Head>
+      <title>Next Challenge | Home</title>
+        <meta name="description" content="Challenge to Next Js generated by create next app" />
+        <link rel="icon" href="/favicon.ico" />
```

Comment:
You can add the favicon on the _App.js instead of adding it on every page

Reply:
I Have a problem with this